hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationBackupHandler.java

```diff
@@ -188,14 +231,21 @@ private Data getBackupOperationData(BackupAwareOperation backupAwareOp) {
         // if getServiceName() method is overridden to return the same name
         // then this will have no effect.
         backupOp.setServiceName(op.getServiceName());
-        return nodeEngine.getSerializationService().toData(backupOp);
+        return backupOp;
     }
 
-    private Backup newBackup(BackupAwareOperation backupAwareOp, Data backupOpData, long[] replicaVersions,
+    private Backup newBackup(BackupAwareOperation backupAwareOp, Object backupOp, long[] replicaVersions,
             int replicaIndex, boolean respondBack) {
-
         Operation op = (Operation) backupAwareOp;
-        Backup backup = new Backup(backupOpData, op.getCallerAddress(), replicaVersions, respondBack);
+        Backup backup;
+        if (backupOp instanceof Operation) {
```

Comment:
hard to read these namings `backup` and `backupOp` what about `backupWrapper` instead of `backup` or some other name more clear?